config/database.yml

```diff
@@ -24,7 +24,8 @@ default: &default
 development:
   <<: *default
   database: webpage-versions-db_development
-
+  password: postgres
+  #password: Figaro.env.LOGINROLE_PASSWORD
```

Comment:
I think you’ve got two options here.

1. You can use ERB syntax to pull in dynamic values:

    ```yml
    password: <%= Figaro.env.LOGINROLE_PASSWORD %>
    ```
  
    …although I think a name that clarifies this is the *database* password might be an improvement over `LOGINROLE_PASSWORD`.

2. Instead of setting a `LOGINROLE_PASSWORD` environment var, set `DATABASE_URL`, which is a standard name ActiveRecord will use automatically instead of the configuration in `database.yml`:

    ```yml
    # in config/application.yml
    development:
      DATABASE_URL: 'postgres://username:password@localhost:5432/database_name'
    ```